rust/agama-cli/src/logs.rs

```diff
@@ -45,6 +51,42 @@ pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
     }
 }
 
+// Whatewer passed in dest formed into an absolute path with archive name
+// if dest is none then a default is returned
```

Comment:
For documenting function arguments, it is common [to use a list](https://doc.rust-lang.org/rust-by-example/meta/doc.html)

```
* `dest`: ...
```

Reply:
tried to modify